app/views/posts/index.html.erb

```diff
@@ -0,0 +1,61 @@
+<% @title='My Blog | Posts' %>
+<div class='mt-3 d-flex align-items-center flex-column'>
+  <div class="card mb-3 border-primary" style="width: 680px;">
+    <div class="row g-0">
+      <div class="col-md-3">
+        <img src="<%= @user.photo %>" class="img-fluid rounded-start" alt="...">
+      </div>
+      <div class="col-md-4">
+        <div class="card-body">
+          <h2 class="card-title"><%= @user.name %></h2>
+        </div>
+        <div class='d-flex justify-content-end' style="width: 350px;">
+          <p class="card-text"><small class="text-muted">Number of posts: <%= @user.posts_counter %></small></p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <h4>Recent Posts</h4>
+  <% if @user.most_recent_posts.empty? %>
+    <div class="card mb-3 border-danger" style="width: 540px;">
+      <div class="card-header bg-danger text-white">No Post Found</div>
+      <div class="card-body">
+        <h5 class="card-title text-danger">Please first submit a post then come back to this page thanks.</h5>
+      </div>
+    </div>
+  <% else %>
+    <% @user.most_recent_posts.each do |post| %>
```

Comment:
- [ ] Kindly display all posts inside the `Post#index` instead of `most_recent_posts` to match the wireframe.